azkaban-exec-server/src/main/java/azkaban/execapp/AzkabanExecutorServer.java

```diff
@@ -486,7 +497,6 @@ private void configureMBeanServer() {
     logger.info("Registering MBeans...");
     mbeanServer = ManagementFactory.getPlatformMBeanServer();
 
-    registerMbean("executorJetty", new JmxJettyServer(server));
```

Comment:
I don't think this class is used anywhere. Have you seen any usages of this class ?
That's why I removed it. Also, the class works on internal APIs of Jetty which are no longer supported or has been changed. I chose to get rid of dead code.